build.gradle

```diff
@@ -281,21 +217,13 @@ subprojects { project ->
             from("${buildDir}/publications/mavenJava") {
                 include 'pom-default.xml.asc'
                 include 'pom-default.xml'
-                rename 'pom-default.xml.asc', "${project.name}-${project.version}.pom.asc"
-                rename 'pom-default.xml', "${project.name}-${project.version}.pom"
+                rename 'pom-default.xml.asc', "${subproject.name}-${subproject.version}.pom.asc"
+                rename 'pom-default.xml', "${subproject.name}-${subproject.version}.pom"
             }
         }
-
     }
 }
 
-task allJars {
-    // Adds as dependency all tasks in sub-projects that
-    // has the type "Jar". This includes, for example, the
-    // integration tests.
-    dependsOn subprojects.collect { it.tasks.withType(Jar) }
```

Comment:
Did you run the integration tests without this task here?

I can't remember why I added this `allJars` task, but by reading the comment I think it might be related to integration tests.

Reply:
The integration tests run successfully without this task being present (just tested).
I removed it because I couldn't find anything that depends on it, also not implicitly  (e.g., an implicit plugin task depending on it).